positional_encodings/tf_positional_encodings.py

```diff
@@ -34,4 +47,8 @@ def call(self, inputs):
         emb_x = tf.tile(emb_x, (1,y,1))
         emb_y = tf.tile(emb_y, (x,1,1))
         emb = tf.concat((emb_x, emb_y),-1)
-        return tf.repeat(emb[None, :, :, :org_channels], tf.shape(inputs)[0], axis=0)
\ No newline at end of file
+        pos_enc = tf.repeat(emb[None, :, :, :org_channels], tf.shape(inputs)[0], axis=0)
+        if self.return_format == 0:
+            return pos_enc
+        elif self.return_format == 1:
+            return inputs + pos_enc
```

Comment:
and then we can just do:

```python
if self.return_format == "sum":
    return inputs + pos_enc
return inputs
```

Reply:
Sure, fine. Both good by me. But I thought that string comparisons are slightly slower than int comparisons but I might be wrong on that.